pinot-segment-local/src/main/java/org/apache/pinot/segment/local/recordtransformer/CompositeTransformer.java

```diff
@@ -69,12 +70,22 @@ public class CompositeTransformer implements RecordTransformer {
    *   </li>
    * </ul>
    */
-  public static CompositeTransformer getDefaultTransformer(TableConfig tableConfig, Schema schema) {
```

Comment:
Let's keep this function to be backward compatible. I feel that there will be code bases that will break from this change. We can add new functions for the new requirement?

Reply:
Makes sense. Restored old function.